registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/validator/property/BankNameValidator.java

```diff
@@ -6,7 +6,7 @@
 import java.util.regex.Pattern;
 
 public class BankNameValidator implements PropertyValidator {
-    private static final String BANK_NAME_REGEX = "^[\\w\\-\\säöüÄÖÜß]+$";
+    private static final String BANK_NAME_REGEX = "^[\\w\\-\\säöüÄÖÜß&·.,()+/'\"]+$";
```

Comment:
Regex can be simplified to this `^[\w\s\WäöüÄÖÜß]+$`
![Screenshot 2019-11-21 at 13 56 58](https://user-images.githubusercontent.com/17544420/69336324-e9fb9d00-0c66-11ea-9154-5fbaa7c32be7.png)

Reply:
It seems to me, that `\w\s\W` will allow any text to pass through, as:
- `\w` stands for `a-zA-Z0-9` and `_` (https://www.w3schools.com/jsref/jsref_regexp_wordchar.asp)
- `\s` stands for whitespaces (https://www.w3schools.com/jsref/jsref_regexp_whitespace.asp)
- `\W` stands for non-word characters (means everything, except `\w`) (https://www.w3schools.com/jsref/jsref_regexp_wordchar_non.asp).